lib/chai/utils/proxify.js

```diff
@@ -32,6 +39,12 @@ module.exports = function proxify (obj) {
       if (typeof property === 'string' &&
           config.proxyExcludedKeys.indexOf(property) === -1 &&
           !Reflect.has(target, property)) {
+        // Special message for invalid property access of non-chainable methods.
+        if (nonChainableMethodName) {
+          throw Error('Invalid use of Chai method: ' + nonChainableMethodName +
+            '. Invocation required.');
+        }
```

Comment:
This error message is a bit unfriendly, I feel like it doesn't explicitly point out where the user went wrong. Could we instead have something like `Invalid Chai property equal.pizza. Did you mean equal(pizza)?`